### PR TITLE
A bunch of updates

### PR DESCRIPTION
**Pull request by AndersonTorres**
A big chunk of updates

- 3mux: 0.3.0 -> 1.0.1
- a2jmidid: 8 -> 9
- stella: 6.2.1 -> 6.3
- pari: 2.11.3 -> 2.11.4
- gp2c: 0.0.11pl3 -> 0.0.11pl4
- guile-reader: 0.6.2 -> 0.6.3
- comic-neue: 2.5 -> 2.51
- kid3: 3.8.3 -> 3.8.4
- luakit: 2.2 -> 2.2.1
- icewm: 1.7.0 -> 1.8.3

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/science/math/pari/default.nix [hunk 1]
```diff
@@ -82,7 +72,6 @@ stdenv.mkDerivation rec {
            of the GP language.
     '';
     homepage    = "http://pari.math.u-bordeaux.fr";
-    downloadPage = "http://pari.math.u-bordeaux.fr/download.html";
```
Comment (7c6f434c) [inline pkgs/applications/science/math/pari/default.nix:85]: Why? I think homepage doesn't have direct tarball link, and the download page does…

**Comment by 7c6f434c** [comment]
Generally looks fine

File: pkgs/applications/science/math/pari/default.nix (hunk 1 above)
Comment (AndersonTorres) [reply pkgs/applications/science/math/pari/default.nix:85]: Well, I think it is a bit superfluous, and we don't use "updaters" here. But yes, on the other hand the direct link is not so obvious.

**Comment by aaronjanse** [comment]
3mux 1.0.1 is a pre-release, not meant to be packaged yet. Some changes are upcoming that will significantly improve 3mux performance and reliability.

In the future I'll avoid tagging pre-releases, since they seem to be picked up by a lot of package management machinery :-)

**Comment by aaronjanse** [comment]
Regardless, thank you @AndersonTorres for all the work done in this PR! I don't know enough about the other packages to review them, but I appreciate your work

**Comment by AndersonTorres** [comment]
@aaronjanse well, I just followed Repology. And the jump from 0.3 to 1.0 looked high enough to not be ignored. Anyway, many thanks to tell me!

**Comment by AndersonTorres** [comment]
Well, there are only two packages from other people, namely 3mux and a2jmidid. The maintainer of a2jmidid, @goibhniu ,  looks stalled. I will merge it, then.

File: pkgs/applications/science/math/pari/gp2c.nix [hunk 2]
```diff
@@ -19,9 +18,8 @@ stdenv.mkDerivation rec {
 
   meta = with stdenv.lib; {
     description =  "A compiler to translate GP scripts to PARI programs";
-    homepage    = "http://pari.math.u-bordeaux.fr/";
-    downloadPage = "http://pari.math.u-bordeaux.fr/download.html";
```
Comment (7c6f434c) [inline pkgs/applications/science/math/pari/gp2c.nix:23]: I think this downloadPage could also be kept then.

**Review by timokau — APPROVED** [review]
I personally would have left out the re-wrapping of documentation lines and I disagree that the `repo = pname` changes are an improvement, but there's no real consensus there and it's not important.

Looks good to me. Thanks!

**Pull request merged**